### ISSUE TITLE
Keywords for Symbols

### ISSUE THREAD

**Issue by ninapaley** [toolness/mystic-symbolic#51]
Related to #16 and #36: I should put keywords in symbols' TOML files. Rather than having to go through every symbol name in the vocabulary, the user could select from a smaller list of keywords/categories. For example, "Love" could include heart.svg, bed.svg and snake_helix.svg. "Death" could include skull.svg, coffin.svg, sword.svg, and bone.svg.

@toolness, how would I express this in TOMLs? I'm thinking something like:
`keyword = ["Love", "Death", "Nature", "Conflict", "Animals"]`

Perhaps y'all could suggest keywords for this list. Off the top of my head:
Love
Death
Nature
Conflict
Animals
Relationships
Sex
Wealth
Learning
Music
Water
Air
Fire
Earth
Food
Women
Men
Risk/Chance/Fate/Fortune
Freedom
State
Home
Strength
Protection
Plants
Religion
Birth/Renewal

Maybe the list should be shorter? I'll read up on Tarot cards and see what sort of categories they keep repeating.

In Tarot readings, the user thinks of, but doesn't say aloud, a question. A user could do the same for MysticSymbolic in "oracle mode," not specifically writing a question, but choosing a keyword category or two. The code would then pick from symbols containing those keywords, plus some "universal" or other symbols to mix things up. Maybe "Universal" could be a keyword tag, too, for symbols like triskele.svg for example, which could be marked:
``keyword = ["Spiral", "Universal"]``

**Comment by toolness** [comment]
Yeah that sounds great--the only thing I'd suggest is making the name `keyword` plural, so instead of

`keyword = ["Spiral", "Universal"]`

it'd be

`keywords = ["Spiral", "Universal"]`

how's that?

**Comment by ninapaley** [comment]
@toolness "keywords" makes mores sense, yes. Thanks!

**Comment by ninapaley** [comment]
At last night's meeting @corinna000 suggested each symbol could have two sets of keywords:

1. Literal/descriptive, i.e. butterfly, wing, insect, animal
2. Symbolic, i.e. freedom, fairy, magic, childhood, spring
